selctionSort.py

```diff
@@ -0,0 +1,23 @@
+# Python program for implementation of Selection 
+# Sort 
+import sys 
+A = [64, 25, 12, 22, 11] 
+
+# Traverse through all array elements 
+for i in range(len(A)): 
+	
+	# Find the minimum element in remaining 
+	# unsorted array 
+	min_idx = i 
+	for j in range(i+1, len(A)): 
+		if A[min_idx] > A[j]: 
+			min_idx = j 
+			
+	# Swap the found minimum element with 
+	# the first element		 
+	A[i], A[min_idx] = A[min_idx], A[i] 
+
+# Driver code to test above 
```

Comment:
Needs more description!

Reply:
Okay, This is a code I wrote it to show how the selection sort works and how it can be implemented in python

Reply:
Haha!